src/server/routes/actions.ts

```diff
@@ -26,19 +33,83 @@ router.get<unknown, BalanceResponse>("/balance", (_, res) => {
     });
 });
 
+const missingParameterError = (res: Response<DripErrorResponse>, parameter: string): void => {
```

Comment:
Good idea to extract this.
While we're at it, we could enforce stricter types to catch potential mistakes in the future:

```suggestion
const missingParameterError = (res: Response<DripErrorResponse>, parameter: keyof BotDripRequestType | keyof FaucetRequestType): void => {
```

Reply:
Oh I love that! That's such a good idea!